staffmanagement/Program.cs

```diff
@@ -1,34 +1,23 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using Newtonsoft.Json;
-using System.IO;
-using Newtonsoft.Json.Linq;
-
-namespace staffmanagement.DataLayer
+using System.Configuration;
+using System.Reflection;
+using Library;
+
+namespace StaffManagement
 {
     class Program
     {
-        
-
         static void Main(string[] args)
-        {
-            Console.WriteLine("Staff Managment System");
+        { 
+            string dllpath = "C:/Users/admin/Source/Repos/staffmanagement/rckr/Data/bin/Debug/netcoreapp3.1/StaffManagement.DataLayer.dll";
```

Comment:
do not use a hard-coded `dll path`. use assembly-qualified class name instead

Reply:
removed hard coded path